app/registries/foreman/settings.rb

```diff
@@ -104,4 +104,539 @@
       default: nil,
       full_name: N_('Instance title'))
   end
+
+  category(:authentication, N_('Authentication')) do
```

Comment:
This is now a very long list. Isn't it possible to keep it in separate files anymore?

Reply:
It most definitelly is, we can have file per category, it would be more readable for sure :+1: